packages/react-devtools-shared/src/backend/utils.js

```diff
@@ -183,3 +183,11 @@ export function format(
 
   return '' + formatted;
 }
+
+export function isSynchronousXHRSupported(): boolean {
+  return (
+    window.document &&
+    window.document.featurePolicy &&
+    window.document.featurePolicy.allowsFeature('sync-xhr')
+  );
+}
```

Comment:
I'm a little surprised Flow didn't complain about this. If `window.document` or `window.document.featurePolicy` aren't defined, this function will return undefined rather than a boolean.